x-pack/plugins/security_solution/server/graphql/timeline/schema.gql.ts

```diff
@@ -183,8 +183,8 @@ export const timelineSchema = gql`
   }
 
   input PageInfoTimeline {
-    pageIndex: Float!
-    pageSize: Float!
+    pageIndex: Float
```

Comment:
why did we have to change the type?

Reply:
as I want to be able to get all the timelines here x-pack/plugins/security_solution/server/lib/detection_engine/scripts/timelines/get_all_timelines.sh

Reply:
same as here https://github.com/elastic/kibana/pull/67496#discussion_r454337222